### DISCUSSION TITLE
Any plans to adapt to HA Assist Pipelines?

### DISCUSSION THREAD

**Discussion by mattkasa** [toverainc/willow#52] [General]
Hi, I was just curious if given the developments on Home Assistant's [Assist Pipeline](https://www.home-assistant.io/integrations/assist_pipeline), has anyone started looking at using the assist pipeline as the backend to process the audio while using wakenet for the wakeword?

**Comment by NobleWolf** [comment]
@mattkasa these two comments from @kristiankielhofner should answer your question.

https://community.home-assistant.io/t/willow-voice-assistant/572004/17?u=noblewolf

https://arstechnica.com/civis/threads/willow-is-a-faster-self-hosted-diy-voice-assistant-built-on-50-gadgets.1492011/post-41870286

**Comment by kristiankielhofner** [comment]
Thanks for tracking those down @NobleWolf!

While not addressing this specifically the general sentiment remains the same. We want to be very careful and deliberate; there is a tremendous amount going on when it comes to speech/audio within the broader ecosystem and the situation needs to stabilize a bit before we settle on our approach to functionality like that offered with Assist Pipeline.
